src/App.tsx

```diff
@@ -1,61 +1,108 @@
+import { useState } from 'react';
 import './App.scss';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [actualTodos, setActualTodos] = useState<Todo[]>(todosFromServer);
+  const actualUsers: User[] = [...usersFromServer];
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState('0');
+  const [checkUser, setCheckUser] = useState(false);
+  const [checkTitle, setCheckTitle] = useState(false);
+
+  const handleTitleChange = () => {
+    setCheckTitle(false);
+  };
+
+  const handleUserChange = () => {
+    setCheckUser(false);
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    setCheckTitle(!title
+      && !(title.length > 0
+      && title.split(' ').length - 1 === title.length));
```

Comment:
Consider using the trimming function to check the length of the actual title instead of these multiple checks. Also, you can use an additional variable for this to avoid repeating code

Reply:
done